robotbase/src/main/java/frc/robot/RobotContainer.java

```diff
@@ -37,71 +40,74 @@
   // The robot's subsystems and commands are defined here...
   private FalconTrajectoryDriveSubsystem m_driveSub;
+  private final ClimbSubsystem m_climbSub;
+  private final FeederSubsystem m_feederSub;
+  public final IntakeSubsystem m_intakeSub;
   private final ShooterSubsystem m_shootSub;
-  private final IntakeSubsystem m_intakeSub;
   private final StorageSubsystem m_storageSub;
+  private final TurretSubsystem m_turretSub;
   private final TogglableLimelightSubsystem m_visionSub;
-  private final ClimbSubsystem m_climbSub;
-
+  private final Compressor m_compressor;
 
   private final InvertDriveControls m_driverControls;
   private final GunnerControls m_gunnerControls;
-  //public final VL53LOXSensorOutput m_sensor = new VL53LOXSensorOutput(Constants.BAUD_RATE, Port.kUSB);
+  // public final VL53LOXSensorOutput m_sensor = new
+  // VL53LOXSensorOutput(Constants.BAUD_RATE, Port.kUSB);
 
-  private final AutoWaitTimeAndChooser[] m_waitTimeAndChooser = new AutoWaitTimeAndChooser[3];
+  // private final AutoWaitTimeAndChooser[] m_waitTimeAndChooser = new
+  // AutoWaitTimeAndChooser[3];
 
   /**
    * The container for the robot. Contains subsystems, OI devices, and commands.
    */
   public RobotContainer() {
     SubsystemFactory subsystemFactory = new SubsystemFactory();
-    m_driveSub = subsystemFactory.CreateFalconTrajectoryDriveSubsystem();
-    m_shootSub = subsystemFactory.CreateShooterSubsystem();
+    m_driveSub = null;// subsystemFactory.CreateFalconTrajectoryDriveSubsystem();
+    m_climbSub = null;// subsystemFactory.CreateClimbSubsystem();
+    m_feederSub = subsystemFactory.CreateFeederSubsystem();
     m_intakeSub = subsystemFactory.CreateIntakeSubsystem();
+    m_shootSub = subsystemFactory.CreateShooterSubsystem();
     m_storageSub = subsystemFactory.CreateStorageSubsystem();
-    m_visionSub = subsystemFactory.CreateLimelightSubsystem();
-    m_climbSub = subsystemFactory.CreateClimbSubsystem();
+    m_turretSub = subsystemFactory.CreateTurretSubsystem();
+    m_visionSub = null;// subsystemFactory.CreateLimelightSubsystem();
+    m_compressor = new Compressor();
+    // m_compressor.setClosedLoopControl(false);
 
     // Configure the button bindings
     m_driverControls = new InvertDriveControls.InvertDriveControlsBuilder(new XboxController(0), .1)
-                      .withDriveSub(m_driveSub)
-                      .withVisionSub(m_visionSub)
-                      .build();
+        .withDriveSub(m_driveSub).withVisionSub(m_visionSub).build();
 
-    m_gunnerControls = new GunnerControls.GunnerControlsBuilder(new XboxController(1))
-                      .withIntakeSub(m_intakeSub)
-                      .withShooterSubsystem(m_shootSub)
-                      .withClimbSubsystem(m_climbSub)
-                      .withStorageSubsystem(m_storageSub)
-                      .build();
+    m_gunnerControls = new GunnerControls.GunnerControlsBuilder(new XboxController(1)).withIntakeSub(m_intakeSub)
```

Comment:
Is there a reason to move these all in line?  It's much harder to read than having each with on a separate line.